_authors/collins-steven.md

```diff
@@ -1,3 +1,7 @@
 ---
 title: "Steven Collins"
 ---
+Steven Collins, PhD, was an academic,trained at Oxford University, where he took Honour Moderations in Literae Humaniores. He taught at Bristol University (England, 1980-87), Indiana University (Bloomington, 1987-89), and Concordia University (Montreal, 1989-91) before joining the University of Chicago. He was a Council Member of the Pali Text Society (London). Professor Collins passed in 2018.
+
+_~ From [The University of Chicago, voices platform ] (https://voices.uchicago.edu/collinssymposium/more-on-steven-collins/)
```

Comment:
In the future, please note that if you start an italic section with an underscore you **must** also close it with a final underscore and that you **must not** put a space between the `]` and `(` characters in a Markdown link.